plugins/services/src/js/reducers/serviceForm/JSONReducers/EnvironmentVariables.js

```diff
@@ -59,11 +59,13 @@ module.exports = {
   },
 
   JSONParser(state) {
-    if (state.env == null) {
+    if (state.env == null && state.environment == null) {
       return [];
     }
 
-    return Object.keys(state.env).reduce(function(memo, key, index) {
+    const env = state.env || state.environment;
```

Comment:
@orlandohohmeier the original solution utilized separate reducer for Multicontainer. Basically, John nailed it, Pods use `environment` where's apps `env` Since this reducer is being used in both cases we need to check both. We actually should have an integration test for that no idea how has this regression happened. 

But since the solution is trivial, you're right, we split reducers and write some integration tests around 💪